userguide/system.rst

```diff
@@ -2681,13 +2681,20 @@ and described in
    +----------------+----------------+-------------------------------------------------------------------------------------------------------------------------------------------------------+
 
 
-.. warning:: Booting an HA pair with failover disabled causes both
-   nodes to come up in standby mode. The |web-ui| shows an additional
-   :guilabel:`Force Takeover` button which can be used to force that
-   node to take control.
-
-.. tip:: The %brand% version of the :command:`ifconfig` command adds
-   two additional fields to the output to help with failover
-   troubleshooting: :samp:`CriticalGroup{n}` and :samp:`Interlink`.
+**Notes about High Availability and failovers:**
+
+Booting an HA pair with failover disabled causes both nodes to come up
+in standby mode. The |web-ui| shows an additional
+:guilabel:`Force Takeover` button which can be used to force that node
```

Comment:
s/node/storage controllers/

Reply:
Changing "node" terminology to "storage controller" is blocked until the TrueNAS UI is updated to switch "node" to "storage controller"